docker/nginx.conf.template

```diff
@@ -3,7 +3,7 @@ server {
     server_name  mwdb-web;
     root         /usr/share/nginx/html;
 
-    client_max_body_size 50M;
+    client_max_body_size "${NGINX_MAX_UPLOAD_SIZE}";
```

Comment:
are the quotes necessary here?

Reply:
I thought it will be safer when someone provides empty value, but I see that `""` is incorrect value as well.

```
mwdb-web-1  | 2024/03/28 13:27:03 [emerg] 1#1: "client_max_body_size" directive invalid value in /etc/nginx/conf.d/default.conf:6
mwdb-web-1  | nginx: [emerg] "client_max_body_size" directive invalid value in /etc/nginx/conf.d/default.conf:6
```

So yeah, I think these quotes can be removed.